axelor-base/src/main/resources/data-init/input/meta_schedule.csv

```diff
@@ -1,2 +1,4 @@
-"active";"name";"job";"cron"
+"active";"name";"job";"cron";"batchServiceSelect";"batchCode"
 "false";"Fetch currency conversion rates on every 5 minutes ";"com.axelor.apps.base.job.CurrencyConversionJob";"0 0/5 * * * ?"
+"false";"Update Customer Account on every day at 1 am ";"com.axelor.apps.base.job.BatchJob";"0 0 1 1/1 * ? *";"com.axelor.apps.account.service.batch.AccountingBatchService";"AX_UPDATE"
```

Comment:
Move this data to axelor-account module
If meta_schedule.csv is not there create new and also update in input-config.xml